scripts/aws/deploy_repo_to_prod_env_api_gw.sh

```diff
@@ -0,0 +1,21 @@
+# Calls deploy.sh script with command line arguments tailored to the stage environment
+#   each deployment is to a single (new) AWS stage
+#  8/28/2023 Created
+#  8/28/2023   new stage = Cerulean, yincludes Studio content host variable,
+
+# Usage (assumes working directory to be the api-manager repo's base folder)
+# ./scripts/aws/deploy_repo_to_prod_env_api_gw.sh <aws stage name>
+
+python3 ./scripts/aws/deploy.py \
+  --tag  None
+  --api-base-domain api.edx.org  \
+  --swagger-filename ./swagger/api.yaml \
+  --landing-page https://stage.edx.org \
+  --rotation-order $1 \
+  --edxapp-host courses.stage.edx.org \
+  --enterprise-host enterprise-catalog.stage.edx.org \
+  --analytics-api-host stage-edx-analyticsapi.edx.org \
+  --registrar-host stage-edx-registrar.edx.org \
+  --catalog-host catalog.stage.edx.org \
+  --enterprise-catalog-host enterprise-catalog.stage.edx.org \
+  --studio-host courses.stage.edx.org
```

Comment:
Do you think it would be better if these variable were read from configuration objects / files that can be updated via PR in the repo, instead of always updating the values in these deploy commands directly?

Reply:
Not certain if that was a good suggestion by me.

Reply:
I'm uncertain what sort of reception this proposal will get from the SRE team. I'll leave anything more than trivial changes for the future, when it's become clear this repo will endure som